tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/ManifestFilters.scala

```diff
@@ -1,7 +1,5 @@
 package org.scalajs.core.tools.jsdep
 
-import scala.collection.immutable.Traversable
```

Comment:
The deletion of this `import` changes the meaning of `Traversable` in this file. Previously it was `collection.immutable.Traversable`, now it's `collection.Traversable`. Is this intentional?